solution/backend/resources/templates/admin/import_resources.html

```diff
@@ -0,0 +1,36 @@
+{% extends "admin/base_site.html" %}
+
+{% block content %}
+<form action="." method="GET" enctype="multipart/form-data">
+    {% csrf_token %}
+    <p>The csv is required to have the following fields in order:</p>
+    <table>
+        <th>Name</th>
+        <th>Description</th>
+        <th>Url</th>
+        <th>Category</th>
+        <th>locations</th>
+    </table><br>
+    <p>You can get a copy of the template by clicking get template. </p>
+    <p><input type="submit" name="get_template" value="Get template"/>
+    <p> CSV's seperate fields by commas.  If you use google sheets or excel to make the csv you dont need to worry but if you use a text editor have the format be like below:</p>
```

Comment:
Recommend capitalizing Google Sheets and Excel, capitalizing CSV and add apostrophe to "don't"

Reply:
done